tests/e2e/lib.sh

```diff
@@ -325,6 +328,13 @@ deploy_sensor() {
 deploy_sensor_via_operator() {
     local sensor_namespace=${1:-stackrox}
     local central_namespace=${2:-stackrox}
+    local central_namespace=${1:-stackrox}
+    info "Deploying sensor via operator into namespace ${sensor_namespace}"
+    if ! kubectl get ns "${sensor_namespace}" >/dev/null 2>&1; then
```

Comment:
If this fails for some transient reason rather than lack of namespace, the next command will be confused, but lack of retries is a general problem in these scripts so meh.

Reply:
Got it. Maybe introduce a kubectl wrapper that implements some retry heuristics? Not as part of this PR, of course.

Reply:
+1. Unfortunately [teaching kubectl to retry is not an option](https://github.com/kubernetes/kubectl/issues/1344).